CONTRIBUTING.md

```diff
@@ -65,32 +65,74 @@ For more information about virtual environment see [here](https://docs.python.or
     python -m venv ./venv
     ```
 
+    OR just using our Makefile
+
+    ```
+    make venv
+    ```
+
 3. Activate the venv
+    
     ```
     source .venv/bin/activate
     ```
+    
+    OR just using our Makefile
+    
+    ```
+    make activate_venv
```

Comment:
https://github.com/wise-agents/wise-agents/issues/192 tracks adding this target to the Makefile.